app/models/user.rb

```diff
@@ -3,27 +3,59 @@ class User < ApplicationRecord
   NAME_LENGTH_MAX = 50
   EMAIL_LENGTH_MAX = 255
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d-]+(\.[a-z\d-]+)*\.[a-z]+\z/i
-  USER_PERMIT = %i(name email password password_confirmation birthday gender
-avatar).freeze
+  USER_PERMIT = %i(name email password password_confirmation birthday gender avatar).freeze
+
+  attr_accessor :remember_token
+
   has_one_attached :avatar
   has_secure_password
   enum gender: {female: 0, male: 1, other: 2}
-  before_save{email.downcase!}
-  scope :newest, ->{order(created_at: :desc)}
+
+  before_save { email.downcase! }
```

Comment:
```suggestion
  before_save {email.downcase!}
```
E đặt logic này ra hàm downcase_email rồi gọi lại nha